### DISCUSSION TITLE
Abogabot

### DISCUSSION THREAD

**Discussion by NestorMyrie** [Launch-X-Latam/MisionFrontEnd#118] [Q&A]
Buenas viajeros , denuevo me surgio una duda.

Cuando realicemos lo solicitado en [la primera práctica](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/01%20-%20INTRO/practicas) como debemos entregar los resultados a el Mission comander? y como se evalúa?

Por ahí lei a algunos de ustedes diciendo que **hay que subir el diseño y Wireframes a un repo de git** pero no le veo mucho sentido siendo el diseño en figma.  y si si es asi a quien debo enviar mi repo?

¡Gracias!

#117 #116

**Comment by ghost** [comment]
creo que el hecho de subir lo desarrollado a tu repo es suficiente siendo que sos parte de un grupo y el staff tiene acceso a ver los repos. Ya han aclarado que no habra evaluaciones pero si miraran los trabajos conforme vayamos progresando.
Suerte!

**Comment by LuceroLuciano** [comment]
Hello!

Puedes subir los links de tus diseños UX/UI a tu repositorio de github para que puedan ser visibles por alguien mas, todo eso es parte de tu portafolio en github.

Saludos.

**Comment by doguedogue** [comment]
Subelos a github, crea un repositorio para esa práctica. Luego te piden la URL para mostrar que lo hiciste.
